<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerForm</class>
 <widget class="QWidget" name="ProductManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>447</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>360</width>
         <height>383</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="clientFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>I&amp;D</string>
            </property>
            <property name="buddy">
             <cstring>idLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="productnameLabel">
            <property name="text">
             <string>&amp;Product Name</string>
            </property>
            <property name="buddy">
             <cstring>productnameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="productnameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>&amp;Price</string>
            </property>
            <property name="buddy">
             <cstring>priceLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="priceLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string>&amp;Category</string>
            </property>
            <property name="buddy">
             <cstring>categoryLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="categoryLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Quantity</string>
            </property>
            <property name="buddy">
             <cstring>quantityLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="quantityLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>360</width>
         <height>383</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="searchTableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Product Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Category</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>S&amp;earch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
